### PR TITLE
Fetch Data Detail

### PR DESCRIPTION
**Pull request by vitconpc**
get data detail
https://edu-redmine.framgia.vn/issues/1520

### CODE REVIEW

**Comment by vitconpc** [comment]
ready

File: app/src/main/java/vn/com/framgia/movie_db26/data/reponsitory/FilmDetailRepository.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import io.reactivex.Observable;
+import vn.com.framgia.movie_db26.data.model.FilmDetail;
+import vn.com.framgia.movie_db26.data.source.FilmDetailDataSource;
+import vn.com.framgia.movie_db26.data.source.remote.FilmDetailRemoteDataSourceImpl;
+
+public class FilmDetailRepository {
+    private static FilmDetailRepository sInstance;
+    private FilmDetailDataSource.FilmDetailRemoteDataSource mSource;
+
+    public FilmDetailRepository(FilmDetailDataSource.FilmDetailRemoteDataSource source) {
```
Comment (tuannguyen38) [inline app/src/main/java/vn/com/framgia/movie_db26/data/reponsitory/FilmDetailRepository.java]: private constructor

File: app/src/main/java/vn/com/framgia/movie_db26/data/source/remote/FilmDetailRemoteDataSourceImpl.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import vn.com.framgia.movie_db26.utils.service.NameAPI;
+
+public class FilmDetailRemoteDataSourceImpl implements
+        FilmDetailDataSource.FilmDetailRemoteDataSource {
+
+    private static FilmDetailRemoteDataSourceImpl sInstance;
+    private static final String API_KEY = BuildConfig.API_KEY;
+    private NameAPI mNameAPI;
+
+    public FilmDetailRemoteDataSourceImpl(NameAPI nameAPI) {
```
Comment (tuannguyen38) [inline app/src/main/java/vn/com/framgia/movie_db26/data/source/remote/FilmDetailRemoteDataSourceImpl.java]: private constructor

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -19,7 +19,13 @@
...
+
+        </activity>
+        <activity android:name=".screen.youtube.YoutubeActivity"
+            android:configChanges="orientation|screenSize">
+
+        </activity>
     </application>
 
-</manifest>
+</manifest>
```
Comment (tuannguyen38) [inline app/src/main/AndroidManifest.xml]: need an endline

**Pull request merged**